app/webpacker/controllers/admin_only_controller.js

```diff
@@ -0,0 +1,29 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller {
```

Comment:
We need to cut down the initial logic into a new JS controller handling the Admin Only tab.